Documentation/workflow/UsingYourBuild.md

```diff
@@ -1,209 +1,227 @@
+  this DLL.
+* System.Private.CoreLib.dll - If you modified C# it will end up here.
+* System.Private.CoreLib.ni.dll - the native image (code) for System.Private.Corelib.   If you modify C# code
+you will want to update both of these together in the target installation.
+
+Thus after making a change and building, you can simply copy the updated binary from the `bin\Product\<OS>.<arch>.<flavor>`
+directory to your publication directory (e.g. `helloWorld\bin\Debug\netcoreapp2.0\win7-x64\publish`) to quickly
+deploy your new bits. In a lot of cases it is easiest to just copy everything from here to your publication directory.
+
+You can build just the .NET Library part of the build by doing (debug, for release add 'release' qualifier)
+(on Linux / OSX us ./build.sh)
+```bat
+    .\build skiptests skipnative
 ```
-       "Microsoft.NETCore.Runtime.CoreCLR": "1.2.0-beta-24528-0"
+Which builds System.Private.CoreLib.dll AND System.Private.CoreLib.ni.dll (you will always want both) if you modify
+C# code. If you wish to only compile the coreclr.dll you can do
+ ```bat
+    .\build skiptests skipmscorlib
 ```
+Note that this technique does not work on .NET Apps that have not been published (that is you have not created
+a directory with all DLLs needed to run the all)  That is because the runtime is either fetched from the system-wide
+location that dotnet.exe installed, OR it is fetched from the local nuget package cache (which is where your
+build was put when you did a 'dotnet restore' and had a dependency on your particular runtime).    In theory you
+could update these locations in place, but that is not recommended since they are shared more widely.
 
-The differences between the project.json generated by the tool and the replacement:
+## Update CoreCLR using runtime nuget package
 
-- Removed Microsoft.NETCore.App platform dependency (`"type": "platform"`). This tells the build system that you don't want to 
-use runtime and libraries that came with the dotnet.exe tool but to fetch the dependencies from the Nuget cache.  If you don't do this
-the tools will ignore your request to make the app use an explicitly specified runtime.   
-- Added the 'runtimes' line at the top level.  The runtime name includes the OS name and the architecture name
-you can find the appropriate name for your OS [here](https://github.com/dotnet/core-docs/blob/master/docs/core/rid-catalog.md).
-This tells the tools exactly which flavor of OS and processor architecture you are running on, so it can find the right
-Nuget package for the runtime.
-- Changed netcoreapp1.0 to netcoreapp1.1. This tells the tools that you want to use the latest .NET Core Framework.
-- Expanded Microsoft.NETCore.App metapackage into explicit list of the .NET Core Framework packages because of there is no good published
-build of Microsoft.NETCore.App metapackage for netcoreapp1.1 yet.
+In order to use the runtime nuget package you will need to do some further setup steps.
```

Comment:
Add something here on why somebody may want to use the nuget package? Something like:

Updating CoreCLR from raw binary output is easier for quick one-off testing. Using Nuget package is better for referencing your CoreCLR build in your actual application because of it does not require manual copying of files around each time the application is built.